src/checksum.c

```diff
@@ -1155,7 +1159,11 @@ DWORD WINAPI SumThread(void* param)
 			uprintf("Unable to create checksum thread event: %s", WindowsErrorString());
 			goto out;
 		}
+#if _MSC_VER && !__INTEL_COMPILER
```

Comment:
Same remark as above, with a `CREATETHREAD()` macro. Now of course the issue here is that `IndividualSumThread` is dereferenced in one case and not in the other, which I'm not too sure how to sort out in a macro, so we might have to go with an `static __inline` function where we pass `&IndividualSumThread` always and just use `*` of that param for the `CreateThread` version.

Note that if you go for a `static __inline` call here, then I'd prefer you did the same for both calls and used something `BeginThread()` and `EndThread()` for the names.